core/import-shearwater.c

```diff
@@ -13,6 +13,8 @@
 #include "membuffer.h"
 #include "gettext.h"
 
+static int sample_rate = 0;
```

Comment:
Recently I introduced that `parser_state` struct. Could this be moved in there?

Reply:
Will do